activities/models/fan_out.py

```diff
@@ -44,13 +46,16 @@ async def handle_new(cls, instance: "FanOut"):
                         identity=fan_out.identity,
                         post=post,
                     )
-            else:
+            elif not post.visibility != Post.Visibilities.local_only:
```

Comment:
Hmm, can we do this in the PostStates when we make the fan-outs, by removing non-local identities there?

Reply:
I think the filtering in `Post.aget_targets()` makes the checks here redundant. I was being a bit more explicit cautious while still wrapping my brain around the state logic.